src/help.ts

```diff
@@ -2,7 +2,10 @@ declare const BBHELP: any;
 
 import { registerScript } from './register-script';
 
-export class BBHelp {
+export class BBHelpClient {
```

Comment:
This is my fault, but you might as well throw an `abstract` on this class definition, since we never want it to be instantiated. `export static class`...